### PR TITLE
Adding system process memory rule on behalf of Dan Rautio

### PR DESCRIPTION
**Pull request by ipmonk**

### CODE REVIEW

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by gvarao** [comment]
Re run

File: juniper_official/System/sys-proc-ext.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
[the remark this answers is not in the document]
Comment (ipmonk) [reply juniper_official/System/sys-proc-ext.yml]: ci has passed after this commit

[the remark this answers is not in the document]
Comment (ipmonk) [reply juniper_official/System/sys-proc-ext.yml]: fixed by c901e94.

[the remark this answers is not in the document]
Comment (ipmonk) [reply juniper_official/System/sys-proc-ext.yml]: fixed by c901e94

[the remark this answers is not in the document]
Comment (ipmonk) [reply juniper_official/System/sys-proc-ext.yml]: fixed by c901e94

[the remark this answers is not in the document]
Comment (ipmonk) [reply juniper_official/System/sys-proc-ext.yml]: fixed by c901e94

**Pull request merged**